template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Data/ApplicationDbContextFactory.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.Extensions.DependencyInjection;
+using Company.WebApplication1.Models;
+
+namespace Company.WebApplication1.Data
+{
+    public class ApplicationDbContextFactory : IDbContextFactory<ApplicationDbContext>
+    {
+        public ApplicationDbContext Create(DbContextFactoryOptions options)
+            => Program.BuildWebHost(Array.Empty<string>())
```

Comment:
(to match [Program.BuildWebHost](https://github.com/dotnet/templating/blob/rel/vs2017/post-rtw/template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Program.cs#L20-L23))